web/js/codeworld_shared.js

```diff
@@ -554,11 +610,18 @@ function registerStandardHints(successFunc) {
           break;
         }
       }
-      const goodOptions = options.slice(0, numGood);
+
+      const goodOptions = options
+        .slice(0, numGood)
+        .reduce((result, current) => {
```

Comment:
What's the problem you are trying to solve here?

Reply:
This servers no purpose any longer, will remove it.